### PR TITLE
#166816107 Users can fetch article's ratings

### PR DESCRIPTION
**Pull request by tunedev**
#### What does this PR do?
Adds fetch articles rating functionality

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories?
pivotal tracker link: [#166816107](https://www.pivotaltracker.com/story/show/166816107)

#### Screenshots (if appropriate)
NA

#### Questions:
NA

### CODE REVIEW

File: src/controllers/article.controller.js [hunk 1]
```diff
@@ -368,5 +370,26 @@ export default {
...
+  },
+
+  async fetchRatings(request, response) {
+    let { articleId } = request.params;
+    articleId = parseInt(articleId, 10);
+
+    const result = await fetchRatingsService(articleId);
+
+    if (result === `Article with id: ${articleId} does not exist`) {
+      return Helper.failResponse(response, 404, result);
```
Comment (dharmykoya) [inline src/controllers/article.controller.js]: Hi @swisskid95, this check was awesome and necessary. Good work, However you didn't write the test for this check. 
Please do write a test for this edge case.
thank you.

Comment (dharmykoya) [inline src/controllers/article.controller.js:382]: Hi @swisskid95, this check was awesome and necessary. Good work, However you didn't write the test for this check.
Please do write a test for this edge case.
thank you.

Comment (tunedev) [reply src/controllers/article.controller.js:382]: @dharmykoya Thanks for pointing this out, would include the check and update this pull request.
